eZ/Bundle/EzPublishCoreBundle/Tests/ApiLoader/RepositoryConfigurationProviderTest.php

```diff
@@ -71,10 +71,11 @@ public function testGetRepositoryConfigNotSpecifiedRepository()
     }
 
     /**
-     * @expectedException \eZ\Bundle\EzPublishCoreBundle\ApiLoader\Exception\InvalidRepositoryException
      */
     public function testGetRepositoryConfigUndefinedRepository()
     {
+        $this->expectException(\eZ\Bundle\EzPublishCoreBundle\ApiLoader\Exception\InvalidRepositoryException::class);
```

Comment:
@mikadamczyk Can we somehow make a script that will change this into import?
Ping @adamwojs as the go to Expert of parsers ;)

Reply:
For sure this also needs manual work in case of alias conflict. We have 835 FQCN already. I think this could be done as a follow-up, to not block this PR.